attachment_queue/models/attachment_queue.py

```diff
@@ -107,5 +104,10 @@ def set_done(self):
         """
         Manually set to done
         """
-        message = "Manually set to done by %s" % self.env.user.name
-        self.write({"state_message": message, "state": "done"})
```

Comment:
@clementmbr 
Yes, it is right that it is a pity to erase the error message.
But, a user can also click on the button run, in order to process the attachment.
Of from attachment_synchronize, I think there is a button on the task to run the export of the pending attachment. So, it won't always be odoo bot that will process the attachment.

If a user process the attachment, then we have no way to know if it really processed it, or just set to done with no further action.

I prefer to erase the potential error message to indicate it has manually been set to done, so we know for sure.
I think it is important to know for sure because else, we could wonder why nothing happened even if the attachment queue is done...
What do you think?